src/main/java/bellcurvegod/command/ExitCommand.java

```diff
@@ -0,0 +1,27 @@
+package bellcurvegod.command;
+
+import static bellcurvegod.storage.Storage.updateData;
+
+import java.io.IOException;
+
+import bellcurvegod.ui.Ui;
+
+/**
+ * Encapsulates the exitCommand.
+ */
+public class ExitCommand implements Runnable {
+    /**
+     * Says goodbye to the user and exits.
+     */
+    public static void run() {
+        try {
+            updateData();
+        } catch (IOException e) {
+            System.out.println(e);
+        }
+
+        Ui.showLine();
```

Comment:
Have you considered extracting the displaying of messages to your Ui class? Or is there any reason why you chose to do it this way instead?